edweiss-app/__test__/memento/card.test.tsx

```diff
@@ -1,3 +1,13 @@
+/**
+ * @file card.test.tsx
+ * @description Tests for the CardListScreen component, which displays a list of cards in a deck.
+ * @author Tuan Dang Nguyen
+ */
+
+// ------------------------------------------------------------
+// --------------- Import Modules & Components ----------------
```

Comment:
No need to add these 3 lines imo, we already see that the imports are just below. But that is ok :)

Reply:
I followed the format of Adamm because it looks quite nice. But it's true that it is a bit over-kill. I'll see if it is inconvenient in the future, I will delete it.